.github/workflows/build-image-on-pr.yaml

```diff
@@ -1,25 +0,0 @@
-name: Docker build
```

Comment:
What if we build this image in a separate action/PR check and then share it with other workflows (e.g., Playwright) to run it? This would make it very clear that we are verifying the image can be built. As it stands now, you have to know that Playwright is actually building the image. 🤔

Reply:
Agreed, I will see if I can share the images by putting them in the GitHub container registry